.github/workflows/update-contributors.yml

```diff
@@ -0,0 +1,83 @@
+name: allcontributors-auto-detect
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  Update:
+    name: Generate
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Tributors Update      
+        uses: con/tributors@0.0.15
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:        
+
+          # Single text list (space separated) of parsers, leave unset to auto-detect
+          parsers: unset
+
+          # Update lookup with GitHub metadata
+          update_lookup: github
+
+          # Skip these users (example)
+          skip_users: manbat
+
+          # INFO, DEBUG, ERROR, WARNING, etc.
+          log_level: DEBUG
+
+          # If files already exist and an init is done, force overwrite
+          force: true
+
+          # the minimum number of contributions required to add a user
+          threshold: 1
+
+      - name: Checkout New Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BRANCH_AGAINST: "master"
+        run: |
+          printf "GitHub Actor: ${GITHUB_ACTOR}\n"
+          export BRANCH_FROM="contributors/update-$(date '+%Y-%m-%d')"
```

Comment:
so there would be branches for every contributors update?  instructions are due on how to cleanup... ideally should be done automagically (action could remove branches PRs for which were closed, or may be it already does it?)

Reply:
Yes there would be a new branch, and you can have these heads removed automatically in the repository settings. I can enable it for you if you want. The GitHub action isn't responsible for doing that, it's out of scope.

Reply:
some guidance would indeed be appreciated on how to configure that... could probably just go to `con/tributors` docs?

Reply:
Go to your repository settings and look for the checkbox to automatically delete head branches.

Reply:
Ok, did that in Settings in the section "Merge button"

Reply:
Great!